CHANGELOG.md

```diff
@@ -78,6 +78,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- **CUMULUS-3258**
+  - Updated Terraform version from 0.13.6 to 1.5.3. Please see the [instructions to upgrade your deployments](https://github.com/nasa/cumulus/blob/master/docs/upgrade-notes/upgrading-tf-version-1.5.3.md).
+- **CUMULUS-3188**
```

Comment:
Might just be a case of merging main but these entries looks like they're in the wrong spot.

Reply:
Great. I see your commit to bring the 3259 entry under 'Unreleased' but it looks like the PR is also trying to add the below entries to the v16.1.1 release. Does main just need to be merged into this branch?